src/lua/lua_script.rs

```diff
@@ -38,13 +38,17 @@ fn create_default_lua_state(writer: Sender<Event>, dimensions: (u16, u16)) -> Lu
 
             globals.set(GAG_NEXT_TRIGGER_LINE, false)?;
 
-            let json = include_str!("../../resources/lua/json.lua");
-            let lua_json = ctx.load(json).call::<_, rlua::Value>(())?;
+            let lua_json = ctx
+                .load(include_str!("../../resources/lua/json.lua"))
+                .call::<_, rlua::Value>(())?;
             globals.set("json", lua_json)?;
-            let bindings = include_str!("../../resources/lua/bindings.lua");
-            ctx.load(bindings).exec()?;
-            let defaults = include_str!("../../resources/lua/defaults.lua");
-            ctx.load(defaults).exec()?;
+
+            ctx.load(include_str!("../../resources/lua/bindings.lua"))
+                .exec()?;
+            ctx.load(include_str!("../../resources/lua/defaults.lua"))
+                .exec()?;
+            ctx.load(include_str!("../../resources/lua/lua_command.lua"))
+                .exec()?;
```

Comment:
Should probably make this more general and just load all `resources/lua/*.lua`

Reply:
Yup, but that doesn't work with `include_str!()` since that's done at compile. Unless you know something I don't :smile:

Reply:
Yeah, it's not trivial. You'll probably have to make a custom macro for it.
If you prefer, I can do that, too. Something like `lua_load_directory!(ctx: _, path: &str) -> Result<()>` (Don't know the type of context)

Reply:
You are most welcome to give it a shot. This will have to do for the time being.